src/imeta.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include <irods/rods.h>
 #include <irods/rodsClient.h>
+#include <irods/rodsError.h>
```

Comment:
Is this needed?

Reply:
Yes. I had inserted a call to `printErrorStack` without including its header.